README.rst

```diff
@@ -85,6 +85,8 @@ Or with the Arrow client:
 History
 -------
 
+- 4.2.6
+    - Prevent from displaying login/password in the logs
```

Comment:
we should go back and fix this

Reply:
what do you mean? if it's just the changelog, I updated it in #43

Reply:
Perfect!